apps/twap/src/migrations/1688472763977-migration.ts

```diff
@@ -0,0 +1,47 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class Migration1688472763977 implements MigrationInterface {
+    name = 'Migration1688472763977'
```

Comment:
Is this being auto-generated?
Would it be possible to add some more descriptive names or at least some comments that describe what each of these migrations does?

Reply:
They are all autogenerated, but you can edit them. In principle, you can add more info as they are generated, but I would suggest relying on git history on that regard.

Names must always follow a "${name}${timestamp}" fashion though so would be like "SuperInformativeName1231283781".